x-pack/filebeat/module/juniper/srx/test/secintel.log-expected.json

```diff
@@ -0,0 +1,129 @@
+[
+    {
+        "@timestamp": "2016-10-17T13:18:11.618-02:00",
+        "client.ip": "5.196.121.161",
+        "client.port": 1,
+        "destination.ip": "10.10.0.10",
+        "destination.port": 24039,
+        "event.action": "malware_detected",
+        "event.category": [
+            "network",
+            "malware"
+        ],
+        "event.dataset": "juniper.srx",
+        "event.kind": "alert",
+        "event.module": "juniper",
+        "event.original": "category=\"secintel\" sub-category=\"Blacklist\" action=\"BLOCK\" action-detail=\"DROP\" http-host=\"N/A\" threat-severity=\"0\" source-address=\"5.196.121.161\" source-port=\"1\" destination-address=\"10.10.0.10\" destination-port=\"24039\" protocol-id=\"1\" application=\"N/A\" nested-application=\"N/A\" feed-name=\"Tor_Exit_Nodes\" policy-name=\"cc_policy\" profile-name=\"Blacklist\" username=\"N/A\" roles=\"N/A\" session-id-32=\"572564\" source-zone-name=\"Outside\" destination-zone-name=\"DMZ\"",
+        "event.outcome": "success",
+        "event.severity": "14",
+        "event.timezone": "-02:00",
+        "event.type": [
+            "info",
+            "denied",
+            "connection"
+        ],
+        "fileset.name": "srx",
+        "input.type": "log",
+        "juniper.srx.action": "BLOCK",
+        "juniper.srx.action_detail": "DROP",
+        "juniper.srx.category": "secintel",
+        "juniper.srx.feed_name": "Tor_Exit_Nodes",
+        "juniper.srx.policy_name": "cc_policy",
+        "juniper.srx.process": "RT_SECINTEL",
+        "juniper.srx.profile_name": "Blacklist",
+        "juniper.srx.session_id_32": "572564",
+        "juniper.srx.sub_category": "Blacklist",
+        "juniper.srx.tag": "SECINTEL_ACTION_LOG",
+        "juniper.srx.threat_severity": "0",
+        "log.level": "informational",
+        "log.offset": 0,
+        "network.iana_number": "1",
+        "observer.egress.zone": "DMZ",
+        "observer.ingress.zone": "Outside",
+        "observer.name": "SRX-1500",
+        "observer.product": "SRX",
+        "observer.type": "firewall",
+        "observer.vendor": "Juniper",
+        "server.ip": "10.10.0.10",
```

Comment:
We could populate `related.ip` and `related.hosts` for most of these I think

Reply:
Those should be there but seem to be missing for one or two of the ingest pipelines, will review it now and push the relevant pipeline change